tests/parsing_tests.rs

```diff
@@ -133,3 +133,10 @@ fn parse_setup() -> Result<(), ParseError> {
 
     Ok(())
 }
+
+#[test]
+fn parse_xi_get_property_reply() {
+    use x11rb::protocol::xinput::XIGetPropertyReply;
+    let bytes: &[u8] = &[0x1, 0x3b, 0x6b, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
```

Comment:
Are you sure this is a `XIGetPropertyReply`?
What I see is:
- `response_type=1` (this is a reply, great)
- `pad0=0x3b` ????
- `sequence = 0x6b 0x00` (I do not want to think about endian right now, but this could be the 107th request)
- everything else is zero.

The weird value for `pad0` is... well, weird.

Reply:
Okay, XI is even weirder than I thought. The `pad0` field is not actually padding.
https://gitlab.freedesktop.org/xorg/xserver/-/blob/7ae221ad5774756766dc78a73d71f4163ac7b1c6/Xi/xiproperty.c#L1205-1214
Together with this in `/usr/include/X11/extensions/XI2proto.h`:
```c
typedef struct {
    uint8_t     repType;                /**< Input extension major opcode */
    uint8_t     RepType;                /**< Always X_XIGetProperty */
    uint16_t    sequenceNumber;
    uint32_t    length;
    Atom        type;
    uint32_t    bytes_after;
    uint32_t    num_items;
    uint8_t     format;
    uint8_t     pad0;
    uint16_t    pad1;
    uint32_t    pad2;
    uint32_t    pad3;
} xXIGetPropertyReply;
#define sz_xXIGetPropertyReply               32
```
And
```
$ grep X_XIGetProperty /usr/include/X11/extensions/XI2proto.h 
#define X_XIGetProperty                 59
```
Thus, this `pad0` field is not really padding, but just a known constant.

Sorry for the noise.